crates/pg_analyser/CONTRIBUTING.md

```diff
@@ -246,7 +251,7 @@ The documentation needs to adhere to the following rules:
 
 Here's an example of how the documentation could look like:
 
-```rust
+````rust
```

Comment:
will check this, too – my local prettier adds an extra ` for rust

Reply:
Ha, this is actually makes sense, because there are triple backticks *within* the code block. 👍